src/Components/CommentList.jsx

```diff
@@ -34,6 +35,7 @@ const CommentList = ({review_id}) => {
         {comments.map(comment => {
             return <CommentCard key={comment.comment_id} comment={comment} />
         })}
+    <li className="post-comment" key="post-comment"><PostComment setComments={setComments}/></li>
```

Comment:
Is a form really a list item? And would it be at the end of comments? What if there were hundreds.